src/builtins.rs

```diff
@@ -89,10 +89,15 @@ fn arg_get(ctx: &mut Context, args: Args) -> Option<ObjectInstance> {
     let args = ctx.args();
 
     // FIXME: Is this cast valid?
-    let result_string = args
-        .get(idx as usize)
-        .map(|s| s.to_owned())
-        .unwrap_or_default();
+    let result_string = match idx {
+        0 => ctx.path().map_or(std::env::args().nth(0).unwrap(), |s| {
```

Comment:
I agree with clippy :D 
```suggestion
        0 => ctx.path().map_or(std::env::args().next().unwrap(), |s| {
```

Reply:
Fixed, thanks